src/TConsole.cpp

```diff
@@ -66,7 +66,9 @@ TConsole::TConsole(Host* pH, bool isDebugConsole, QWidget* parent)
 , mCommandBgColor(Qt::black)
 , mCommandFgColor(QColor(213, 195, 0))
 , mConsoleName("main")
-, mDisplayFont(QFont("Bitstream Vera Sans Mono", 10, QFont::Normal)) //mDisplayFont( QFont("Monospace", 10, QFont::Courier ) )
+, mDisplayFontName("Bitstream Vera Sans Mono")
```

Comment:
Slightly off-topic - but why do we specify **Bitstream Vera** fonts in some places (and normally bundle it) and then use the (later extended from them) **DejaVu** font family in other places?  Is there some reason we cannot switch to DejaVu throughout?

Reply:
I see no need to, and we ship Bitstream Vera with Mudlet.

Reply:
So should we be using DejaVu fonts or not? It seems to me that we should either use the Bitstream Vera or the DejaVu  ones **throughout** for a *consistent* style... 🙁